src/order/RelayOrder.ts

```diff
@@ -65,31 +65,30 @@ type WitnessInfo = {
   universalRouterCalldata: string;
 };
 
-// TODO: update with new witness types
 const RELAY_WITNESS_TYPES = {
```

Comment:
Ordering doesn't matter here since is an object, doing it by order of appearance in the top level struct